x11-misc/adriconf/adriconf-2.4.ebuild

```diff
@@ -0,0 +1,81 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake desktop
+
+DESCRIPTION="Advanced DRI Configurator"
+HOMEPAGE="https://github.com/jlHertel/adriconf"
+if [[ "${PV}" == 9999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://gitlab.freedesktop.org/mesa/adriconf.git"
+else
+	SRC_URI="https://gitlab.freedesktop.org/mesa/adriconf/-/archive/v${PV}/adriconf-v${PV}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+	S="${WORKDIR}/${PN}-v${PV}"
+fi
+LICENSE="GPL-3"
+SLOT="0"
+
+IUSE="wayland"
+
+RDEPEND="
+	dev-cpp/glibmm:2
+	dev-cpp/gtkmm:3.0
+	dev-cpp/libxmlpp:3.0
+	dev-libs/boost:=
+	dev-libs/glib:2
+	dev-libs/libsigc++:2
+	media-libs/mesa
+	sys-apps/pciutils
+	x11-libs/gtk+:3
+	x11-libs/libdrm
+	x11-libs/libX11
+	wayland? ( media-libs/mesa[egl] )
```

Comment:
This needs to be updated to ```[egl(+)]```, as newest mesa release in tree dropped this use flag and enables this always.